dask/dataframe/groupby.py

```diff
@@ -150,18 +452,7 @@ def __init__(self, df, index=None, slice=None, **kwargs):
         self.obj = df
 
         # grouping key passed via groupby method
-        if (isinstance(index, (DataFrame, Series, Index)) and
-           isinstance(df, DataFrame)):
-
-            if (isinstance(index, Series) and index.name in df.columns and
-               index._name == df[index.name]._name):
-                index = index.name
-            elif (isinstance(index, DataFrame) and
-                  set(index.columns).issubset(df.columns) and
-                  index._name == df[index.columns]._name):
-                index = list(index.columns)
-
-        self.index = index
+        self.index = _normalize_index(df, index)
```

Comment:
Here, I added support for lists of series as a grouper, which was previously not supported. This logic is required to handle `df.groupby([df['a'], df['b']])` consistent with pandas. The recursion in `_normalize_index` is not truly necessary (since lists of lists should not be valid groupers), but it was easiest way to implement it.